doc/classes/Skeleton3D.xml

```diff
@@ -126,18 +126,23 @@
 			<return type="Vector3" />
 			<param index="0" name="bone_idx" type="int" />
 			<description>
+				Returns the pose position of the bone at [param bone_idx].  
+				The returned [Vector3] is in the local coordinate space of the [Skeleton3D] node.
```

Comment:
You need to avoid new lines in the description, unless necessary. Each new line creates a paragraph in the rendered documentation, and in this case that would be excessive. (This applies to all added descriptions).

Reply:
Copy.  Removed new lines from the descriptions.